package.json

```diff
@@ -55,15 +55,15 @@
     "eslint-plugin-react-hooks": "^5.0.0",
     "eslint-plugin-storybook": "^0.11.1",
     "eslint-plugin-testing-library": "^7.0.0",
-    "eslint-plugin-unicorn": "^56.0.0",
+    "eslint-plugin-unicorn": "^56.0.1",
```

Comment:
When used in combination with ESLint v9.15.0 or higher, lint will fail, so you need to update eslint-plugin-unicorn.

see: https://github.com/sindresorhus/eslint-plugin-unicorn/pull/2497